agent/command/git.go

```diff
@@ -199,7 +208,22 @@ func (opts cloneOpts) buildHTTPCloneCommand(forApp bool) ([]string, error) {
 	} else {
 		gitURL = thirdparty.FormGitURL(urlLocation.Host, opts.owner, opts.repo, opts.token)
 	}
-	clone := fmt.Sprintf("git clone %s '%s'", gitURL, opts.dir)
+
+	gitCommand := "git clone"
+	if opts.useScalar {
+		scalarAvailable, err := agentutil.IsGitVersionMinimumForScalar(thirdparty.RequiredScalarGitVersion)
+		if err != nil {
+			return nil, errors.Wrap(err, "checking git version")
+		}
+		if scalarAvailable {
+			gitCommand = "scalar clone --no-src"
```

Comment:
can you add a comment to explain --no-src?

Reply:
done